Content.Server/Automod/AutomodManager.cs

```diff
@@ -0,0 +1,260 @@
+﻿using System.Diagnostics;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Content.Server.Database;
+using Content.Shared.Automod;
+using Content.Shared.Database;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Automod;
+
+public sealed class AutomodManager : IAutomodManager, IPostInjectInit
+{
+    [Dependency] private readonly IPrototypeManager _protoMan = default!;
+    [Dependency] private readonly IEntityManager _entMan = default!;
+    [Dependency] private readonly ILogManager _logMan = default!;
+    [Dependency] private readonly IServerDbManager _db = default!;
+
+    private ISawmill _log = default!;
+
+    // Filters
+    private readonly Dictionary<AutomodTarget, Dictionary<Regex, AutomodFilterDef>> _regexFilters = new();
+
+    #region Initialize
+
+    public void Initialize()
+    {
+        IoCManager.InjectDependencies(this);
```

Comment:
This is a manager. You don't need to do this.

Reply:
Removed [ab2759b](https://github.com/space-wizards/space-station-14/pull/28570/commits/ab2759bb3ef693c21147482d5237acaa6a78bb9c)